src/main/java/com/friendly_machines/intellij/plugins/ideanative2debugger/impl/GdbMiFilter.java

```diff
@@ -28,7 +29,8 @@ public GdbMiFilter(DebugProcess process, @NotNull Project project, GdbOsProcessH
         //myChildOut = childOut;
         //Native2DebugProcess process = Native2DebugProcess.getInstance(myOsProcessHandler);
         myChildOut = childIO;
-        myChildIn = childIO.getProcessInput();
+        // pucgenie: It is 7-bit ASCII per mi3 specification, so... use US_ASCII instead?
```

Comment:
What would US_ASCII do when bit 7 is set in input. Crash?

Reply:
GdbOsProcessHandler's getCharset says what is allowed in Console. See e.g. org.jetbrains.kotlin.console.CommandExecutor#sendCommandToProcess